File: app/src/main/java/com/example/pam_app/view/HomeView.java
package com.example.pam_app.view;

import com.example.pam_app.model.BucketEntry;

import java.util.List;

public interface HomeView {

    void bind(final List<BucketEntry> entryList);
    void onBucketEntryAdded(final BucketEntry bucketEntry);
}

File: app/src/main/java/com/example/pam_app/view/HomeView.kt
package com.example.pam_app.view

import com.example.pam_app.model.BucketEntry
import kotlin.jvm.Synchronized

interface HomeView {
    fun bind(entryList: List<BucketEntry>?)
    fun onBucketEntryAdded(bucketEntry: BucketEntry?)
}